### ISSUE TITLE
Flickering effect on 'apm publish' output

### ISSUE THREAD

**Issue by dapplion** [aragon/aragon-cli#1059]
## 🐛 Bug Report

When running `apm publish` on the Aragon demo app (CounterApp)
```
node ../../aragon-cli/packages/aragon-cli/dist/cli.js apm publish minor CounterApp
```
there's a flickering effect on the `listr` output. This is a known issue when some other function or process prints to stdout frequently which conflicts with the internal renderer of `listr`. I was not able to pin point who is printing to stdout but there's a video that can help identify the set of functions that are running when it happens.

----
![aragon-flickering-apm-publish](https://user-images.githubusercontent.com/35266934/70835450-6e23e900-1dfd-11ea-94e2-8315cf51e62e.gif)
*demo .gif*

----

#### Environment

- OS and OS version: Linux Mint 18.0
- AragonCLI version: On develop `9656a8`

## To Reproduce

Use create-aragon-app to clone the CounterApp. Run the devchain and run
```
node ../../aragon-cli/packages/aragon-cli/dist/cli.js apm publish minor CounterApp
```

## Expected behavior

Normal `listr` output

## Actual Behavior

See video above

**Comment by 0xGabi** [comment]
Solved with #1064

**Comment by dapplion** [comment]
@0xGabi Thank you so much!
